docs: improve structure (#676)
open-energy-transition/open-tyndp: doc/cba.rst

* **(e) Apply MSV:** The ``mu_energy_balance`` time series is written into the ``marginal_cost`` of storage components.
* **(f) Hydro Pinning:** Large hydro reservoirs are pinned to their perfect-foresight state-of-charge values at window boundaries to guide dispatch where duals are near-zero.

Solve
^^^^^^^^^^^^^^^^^^^
The prepared network is solved for the **Reference** baseline and then for each **Project**. Projects are evaluated using either **TOOT** (Take Out One at a Time) or **PINT** (Put IN at a Time) methods.

CBA Indicators
==============

Indicators are computed as the difference in system costs and emissions between the reference and project dispatch solutions.

* **B1: Social Economic Welfare (SEW):** Quantifies the change in total system costs (CAPEX + OPEX).
* **B2: Social costs of CO2 emissions:** Calculates the impact using societal cost assumptions (low, central, high).
* **B3: RES integration costs:** Tracks changes in renewable capacity, generation, and avoided curtailment.
* **B4: Non-direct greenhouse emissions:** Quantifies pollutants (NOx, SO2, PM, etc.) using fuel consumption multipliers.

Configuration
=============

> CBA settings are defined in the ``cba`` section of the configuration file.

Reviewer: ```suggestion
CBA settings are defined in the ``cba`` section of the configuration file. You can refer to
the `configuration <configuration.html>`_ page for a more comprehensive list of available PyPSA-Eur
and Open-TYNDP configuration options.```

Example Collection (``config/scenarios.tyndp.yaml``):

.. code-block:: yaml

    NT-cy1995:
    #  <<: *cba-common
    snapshots:
        start: "1995-01-01"
        end: "1996-01-01"

    atlite:
        default_cutout: europe-1995-sarah3-era5

    cba:
        sb_scenario: NT

    NT-cy2008:
    #  <<: *cba-common
    snapshots:
        start: "2008-01-01"
        end: "2009-01-01"

    atlite:
        default_cutout: europe-2008-sarah3-era5

    cba:
        sb_scenario: NT

    NT-cy2009:
    #  <<: *cba-common
    snapshots:
        start: "2009-01-01"
        end: "2010-01-01"

    atlite:
        default_cutout: europe-2009-sarah3-era5

    cba:
        sb_scenario: NT

    NT-cyears:
      cba:
        scenarios: [NT-cy2009, NT-cy2008, NT-cy1995]

Individual child scenarios (e.g., ``NT-cy2009``) must define their specific ``snapshots``, ``atlite.default_cutout``, and the ``cba.sb_scenario`` used as input.

Running Multiple Years
^^^^^^^^^^^^^^^^^^^^^^
To run a collection like ``NT-cyears``, modify ``run.name`` in ``config/config.tyndp.yaml`` or override it via command line:

.. code-block:: console

>     $ snakemake -call cba --configfile config/config.tyndp.yaml --config run='{"name":"NT-cyears"}'

Reviewer 2: ```suggestion
    $ pixi run tyndp-cba --config run='{"name":"NT-cyears"}'
```

Probably better to change to `pixi` to be consistent with the SB section (and better overall)

Running a Single Climate Year
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Similarly, a single climate year can be run by modifying ``run.name``  in ``config/config.tyndp.yaml`` to the desired scenario (e.g., ``NT-cy2009``) or overriding it via command line:

.. code-block:: console

>     $ snakemake -call cba --configfile config/config.tyndp.yaml --config run='{"name":"NT-cy2009"}'

Reviewer 2: ```suggestion
    $ pixi run tyndp-cba --config run='{"name":"NT-cy2009"}'
```

Configuration
=============

CBA settings are defined in the ``cba`` section of the configuration file. You can refer to
the `configuration <configuration.html>`_ page for a more comprehensive list of available PyPSA-Eur
and Open-TYNDP configuration options.```

Project Selection
-----------------
* ``planning_horizons``: Selects horizons (e.g., 2030, 2040).
* ``projects``: Defines project identifiers (e.g., ``t1-t35``).
* ``cba_scenario_input``: If ``use_presolved`` is true, the workflow retrieves pre-solved SB networks from an archive.

Rolling Horizon Settings
------------------------
* ``storage.cyclic_carriers``: Carriers that remain cyclic within each weekly window.
* ``storage.soc_boundary_carriers``: Carriers pinned at window boundaries.
* ``msv_extraction.resolution``: Controls temporal resolution for the MSV solve (e.g., ``24H``). 

Running Single vs Multiple Climate Years
----------------------------------------

Climate-year collections allow project benefits to be assessed across multiple weather years, consistent with the 2024 TYNDP implementation.

> The CBA entry point ``snakemake -call cba`` can run a **single scenario** that is one single climeate year or a **collection scenario** that defines a list of child (climate years) scenarios under ``cba.scenarios``.

Reviewer 2: ```suggestion
The CBA entry point ``pixi run tyndp-cba`` can run a **single scenario** that is one single climeate year or a **collection scenario** that defines a list of child (climate years) scenarios under ``cba.scenarios``.
```
